File: cfgs/java_cfgs_foo/TOTAL_NUMBER_OF_NON_DECREASING_NUMBERS_WITH_N_DIGITS.java
Nodes
1;special;;;
2;statement;int[][] dp = new int[10][foo_1 + 1];39;
3;statement;int i = 0;40;
4;conditional;foo_1 < 10;41;
5;statement;foo_1[foo_2][1] = 1;42;
6;statement;i++;42;
7;statement;int digit = 0;43;
8;conditional;foo_1 <= 9;44;
9;statement;int len = 2;46;
10;conditional;foo_1 <= foo_2;47;
11;statement;int x = 0;49;
12;conditional;foo_1 <= foo_2;50;
13;statement;foo_1[foo_2][foo_3] += foo_4[foo_5][foo_6 - 1];51;
14;statement;x++;51;
15;statement;len++;48;
16;statement;digit++;45;
17;statement;int count = 0;54;
18;statement;int i = 0;55;
19;conditional;foo_1 < 10;56;
20;statement;count += foo_1[foo_2][foo_3];57;
21;statement;i++;57;
22;statement;return foo_1;58;
23;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;7;
4;6;
5;4;
6;8;
7;9;17;
8;10;
9;11;16;
10;12;
11;13;15;
12;14;
13;12;
14;10;
15;8;
16;18;
17;19;
18;20;22;
19;21;
20;19;
21;23;
22;

File: cfgs/pythoncfgs/TOTAL_NUMBER_OF_NON_DECREASING_NUMBERS_WITH_N_DIGITS.py
Nodes
1;statement;foo_1 = [[0 for foo_2 in fun_1(foo_3 + 1)] for foo_4 in fun_2(10)];8;
2;conditional;fun_1(10);9;
3;conditional;fun_1(10);11;
4;statement;foo_1 = 0;15;
5;conditional;fun_1(10);16;
6;statement;return foo_1;18;
7;statement;foo_1 += foo_2[foo_3][foo_4];17;
8;conditional;foo < foo;12;
9;conditional;fun_1(foo_1 + 1);13;
10;statement;foo_1[foo_2][foo_3] += foo_4[foo_5][foo_6 - 1];14;
11;statement;foo_1[foo_2][1] = 1;10;
12;exit;;
Edges
0;2;
1;3;11;
2;4;8;
3;5;
4;6;7;
5;12;
6;5;
7;3;9;
8;8;10;
9;9;
10;2;
11;
